### PR TITLE
Exception mailer

### PR DESCRIPTION
**Pull request by joemsak**
Dependent on #2

### CODE REVIEW

File: lib/openstax_rescue_from.rb [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-end
-
-require "openstax/rescue_from/configuration"
-require "openstax/rescue_from/version"
+require 'openstax/rescue_from/engine'
+require 'openstax/rescue_from/wrapped_exception'
 
 module OpenStax
   module RescueFrom
+    def self.included(base)
```
Comment (Dantemss) [inline lib/openstax_rescue_from.rb]: I would suggest keeping modules to be included separate from the main gem module.

File: lib/openstax/rescue_from/engine.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'active_support/all'
```
Comment (Dantemss) [inline lib/openstax/rescue_from/engine.rb]: Is this really needed?

File: lib/openstax/rescue_from/engine.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+require "openstax/rescue_from/configuration"
+require "openstax/rescue_from/version"
+
+module OpenStax
+  module RescueFrom
+    class Engine < ::Rails::Engine
+      initializer 'openstax.rescue_from.inflection' do
+        ActiveSupport::Inflector.inflections do |inflect|
+          inflect.acronym 'OpenStax'
```
Comment (Dantemss) [inline lib/openstax/rescue_from/engine.rb]: I could be wrong but I don't think you need this unless using Rails to autoload stuff in the gem. And `openstax_utilities` already does this for you.

Comment (joemsak) [inline lib/openstax/rescue_from/engine.rb]: Confirm with @jpslav as he left this for me :)

File: lib/openstax_rescue_from.rb (hunk 1 above)
Comment (joemsak) [inline lib/openstax_rescue_from.rb]: Fixed on #5

File: lib/openstax/rescue_from/engine.rb (hunk 2 above)
Comment (joemsak) [inline lib/openstax/rescue_from/engine.rb]: Nope! Fixed in #5

**Pull request closed without merging**